### PR TITLE
Added sprint references, fixed some lilnks and cleaned up outdated info

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

File: src/content/pages/faq.mdx [hunk 1]
```diff
@@ -78,16 +81,12 @@ A. After Call for Proposals (CFP) finished, submissions went through two rounds
 
 ## **Q. What support do you give to speakers?**
 
-A. Accepted speakers are eligible for a free Ticket; more details about the ticket types that speakers are eligible for will be announced soon. In addition, speakers are encouraged to apply for additional [Financial Aid](/finaid), should there be a need.
+A. Accepted speakers are eligible for a free ticket. Detailed information regarding the types of tickets available to speakers has been sent directly via email. In addition, speakers are encouraged to apply for additional [Financial Aid](/finaid), should there be a need.
```
Comment (cybit) [inline src/content/pages/faq.mdx:84]:
```suggestion
A. Accepted speakers are eligible for a free ticket. Detailed information regarding the types of tickets available to speakers has been sent directly via email. In addition, speakers were encouraged to apply for additional [Financial Aid](/finaid), if needed.
```

Comment (cybit) [reply src/content/pages/faq.mdx:84]: finaid is finished I think

File: src/content/pages/faq.mdx [hunk 2]
```diff
@@ -102,11 +101,11 @@ A. Yes, in-person participation applications for our [Financial Aid Programme](/
 
 ## **Q. When can sponsors start signing up?**
 
-A: The 2024 packages are not finalised yet. Although, more details will be announced soon!
+A. We are open for sponsorship signup, though availability for some packages may be limited or pending. You can find all the details of the 2024 sponsorship packages and options on our [sponsorship page](/sponsor), as well as the demographics of previous years.
 
-**Sponsors who sign up early will again receive a 10% discount.**
+**Sponsors who sign up by 15th of April again received a 10% discount.**
```
Comment (cybit) [inline src/content/pages/faq.mdx:106]:
```suggestion
**Sponsors who signed up by 15th of April again received a 10% discount.**
```

**Review by cybit — COMMENTED** [review]
Wow. Always amazed what you find in the pages.

**Comment by hypha** [comment]
@egeakman I realised that if this is merged, then links on [this blogpost](https://blog.europython.eu/maximize-your-experience-at-europython-2024/) (maybe more?) will be broken now... :sob: Maybe I put the mdx files back but just don't link it on the website...?

**Comment by egeakman** [comment]
@hypha We can create Astro pages that redirect to session pages to avoid broken links on dependents.

The main concern I have right now is the maximum abstract length on Pretalx is 1700, and it counts line breaks as ``\r\n``, so I'm having trouble copying the contents to Pretalx :(

**Comment by hypha** [comment]
> @hypha We can create Astro pages that redirect to session pages to avoid broken links on dependents.

@egeakman Fantastic! Thank you! Feel free to commit to this PR yourself if it is easier, or let me know what to do.  

> The main concern I have right now is the maximum abstract length on Pretalx is 1700, and it counts line breaks as `\r\n`, so I'm having trouble copying the contents to Pretalx :(

I am assuming the longer descriptions section doesn't show on the website? A workaround I used often before is to set the abstract max length for longer, update the two pages, and set it back to 1700 char :ninja: 

P.S. I think I accidentally edited your original comment instead of commenting. Sorry! I didn't know this kind of power existed.

**Comment by egeakman** [comment]
@hypha Ahaha no worries :smile:

I changed the maximum length to 2100, and @cmaureir said it's OK to leave it like this.

File: src/content/pages/programme.mdx [hunk 3]
```diff
@@ -20,13 +20,13 @@ A week of all things Python:
...
 
-We start at 9:00 a.m. at the [PCC](/venue). On the first two days, all activities will take place on the first floor only.
+We start at 9:00 a.m. at the [PCC](/where#prague-congress-centre---conference-venue). On the first two days, all activities will take place on the first floor only.
 
 We are hosting [16 tutorials](/tutorials), and two special beginner workshops: [Django Girls](https://djangogirls.org/en/prague/) on Monday and [Humble Data](https://humbledata.org/) on Tuesday.
 
 PyBurger is happening on Monday! We will all meet up at Burgerman & enjoy one of the best burgers in Europe.
 
-Two Summits – [C-API Summit](/c-api-summit) on Monday, and [WASM Summit](/wasm-summit) on Tuesday.
+Two Summits – [C-API Summit](/session/c-api-summit) on Monday, and [WASM Summit](/session/wasm-summit) on Tuesday.
```
Comment (egeakman) [inline src/content/pages/programme.mdx:29]:
```suggestion
Two Summits – [C-API Summit](/session/c-api-summit) on Monday, and [WASM Summit](/session/webassembly-summit) on Tuesday.
```

File: src/data/links.json [hunk 4]
```diff
@@ -54,15 +54,15 @@
...
+          "path": "/session/c-api-summit"
         },
         {
           "name": "Humble Data",
           "path": "/humble-data"
         },
         {
           "name": "WebAssembly Summit",
-          "path": "/wasm"
+          "path": "/session/wasm-summit"
```
Comment (egeakman) [inline src/data/links.json:65]:
```suggestion
          "path": "/session/webassembly-summit"
```

**Review by egeakman — COMMENTED** [review]
``/sesion/wasm-summit`` is changed to ``/session/webassembly-summit``

File: src/components/hero-section/hero-venue.astro [hunk 5]
```diff
@@ -31,16 +31,23 @@ import venueImage from "./venue-hero.jpg";
...
     </Prose>
 
-    <div class="space-x-4" style={{ marginBottom: 10 }}>
+    <div class="space-x-4 mb-5">
       <ButtonLink href="/where#prague-congress-centre---conference-venue">
         PCC Conference Venue
       </ButtonLink>
-
       <Link
         href="https://www.google.com/maps?ll=50.062576,14.429021&z=14&t=m&mapclient=embed&cid=5559277398893748727"
```
Comment (egeakman) [inline src/components/hero-section/hero-venue.astro:39]:
```suggestion
        href="https://maps.app.goo.gl/jXtn7oXVg6vECFNP9"
```

Let's shorten this

Comment (hypha) [reply src/components/hero-section/hero-venue.astro:39]: Hmm now thinking about it again, it it better to link directly to Entrance 5? https://goo.gl/maps/S13ZYR1ktN7ebwyP7

Comment (egeakman) [reply src/components/hero-section/hero-venue.astro:39]: Yes, makes better sense. Thank you!

Comment (egeakman) [reply src/components/hero-section/hero-venue.astro:39]: @hypha updated the suggestion

Comment (hypha) [reply src/components/hero-section/hero-venue.astro]: Oops,I wish I waited.  I just did it here: https://github.com/EuroPython/website/pull/762/commits/ebba04c38400840c183c0aa714b3f0759e582fab. Thank you so much though!

**Comment by egeakman** [comment]
Redirections are done. It now redirects:

- ``/wasm`` -> ``/session/webassembly-summit``
- ``/capi`` -> ``/session/c-api-summit``

**Comment by egeakman** [comment]
> Thank you so much @egeakman! Now I learned how to do that too! I am happy with it. Feel free to merge any time you see fit <3

@hypha The situation we had with WebAssembly Summit and C API Summit pages also exists with Humble Data. But maybe we can handle that in another pull request.

**Review by egeakman — APPROVED** [review]

**Comment by egeakman** [comment]
> (I understand that coordination can be a bit of a burden sometimes) 😊

Coordination with you is never a burden! Thanks again for the PR!

**Pull request merged**